### PR TITLE
增加了量化脚本，swift和xinference的推理readme，在readme中增加了常用模块和新模块的快速导航

### PR DESCRIPTION
**Pull request by LDLINGLINGLING**

### CODE REVIEW

**Review by yiranyyu — CHANGES_REQUESTED** [review]
* Typos shall be fixed before merging (such as `finetuen`).
* First letter should be capital in table titles (such as `inference => Inference` and `quantize` => `Quantization`
* Terminology names shall follow common usage (such as `LoRA` and `Gradio`)
* Section descriptions are required.
* Section titles should follow the format of existing titles, including documents in docs/ folder.
*

**Comment by LDLINGLINGLING** [comment]
非常感谢，已经按照要求修改，如果还有哪里不对，请您指出

**Review by yiranyyu — CHANGES_REQUESTED** [review]
* Specialized terms like LoRA, SWIFT and APK should follow the origin cases (rather than lora or Lora)
* Such two navigation tables still lacks description in their sections.
* Usage of indentation in code blocks in markdown files should be consistent (such as when do we need a tab for leading spaces).

**Comment by LDLINGLINGLING** [comment]
按照要求做了修改，其中apk的专业术语没有修改，这是因为apk在之前的readme中都是小写英文，因此沿用了原来的表述方式

**Review by yiranyyu — CHANGES_REQUESTED** [review]
* README.md and README_zh.md should be consistent. I found mismatches in the navigation table.

Other contents seems to be fine for now.

**Comment by LDLINGLINGLING** [comment]
Ensure that there is no difference between the readme_zh and the readme navigation bar in new commit
.

File: docs/swift_train_and_infer.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+Here are the steps to launch SWIFT from the Bash command line:
+
+1. Run the bash code will download the model of MiniCPM-Llama3-V-2_5 and run the inference
+``` shell
+CUDA_VISIBLE_DEVICES=0 swift infer --model_type minicpm-v-v2_5-chat
+```
+
+2. You can also run the code with more arguments below to run the inference:
+``` 
+model_id_or_path # 可以写huggingface的模型id或者本地模型地址
```
Comment (iceflame89) [inline docs/swift_train_and_infer.md:24]: 建议将注释改都成英文

File: quantize/bnb_quantize.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import torch
+import GPUtil
+import os
+
+model_path = '/root/ld/ld_model_pretrain/MiniCPM-Llama3-V-2_5' # 模型下载地址
+device = 'cuda' if torch.cuda.is_available() else 'cpu'
+save_path = '/root/ld/ld_model_pretrain/MiniCPM-Llama3-V-2_5_int4' # 量化模型保存地址
+image_path = './assets/airplane.jpeg'
+
+# 创建一个配置对象来指定量化参数
```
Comment (iceflame89) [inline quantize/bnb_quantize.py:26]: 注释改成英文

File: quantize/bnb_quantize.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    image=Image.open(image_path).convert("RGB"),
+    msgs=[
+        {
+            "role": "user",
+            "content": "这张图片中有什么?"
+        }
+    ],
+    tokenizer=tokenizer
+) # 模型推理
+print('量化后输出',response)
```
Comment (iceflame89) [inline quantize/bnb_quantize.py:60]: print改成英文

File: quantize/bnb_quantize.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+import torch
+from transformers import AutoModel, AutoTokenizer, BitsAndBytesConfig
+from PIL import Image
+import time
+import torch
+import GPUtil
+import os
+
+model_path = '/root/ld/ld_model_pretrain/MiniCPM-Llama3-V-2_5' # 模型下载地址
+device = 'cuda' if torch.cuda.is_available() else 'cpu'
```
Comment (iceflame89) [inline quantize/bnb_quantize.py:22]: BitsAndBytes 支持cpu吗？

**Review by iceflame89 — CHANGES_REQUESTED** [review]

File: quantize/bnb_quantize.py (hunk 4 above)
Comment (LDLINGLINGLING) [reply quantize/bnb_quantize.py:22]: 这里确实有问题，支持量化，但是不支持推理

File: docs/swift_train_and_infer.md (hunk 1 above)
Comment (LDLINGLINGLING) [reply docs/swift_train_and_infer.md]: 已经修改

File: quantize/bnb_quantize.py (hunk 2 above)
Comment (LDLINGLINGLING) [reply quantize/bnb_quantize.py]: 已经修改

File: quantize/bnb_quantize.py (hunk 3 above)
Comment (LDLINGLINGLING) [reply quantize/bnb_quantize.py]: 已经修改

**Review by iceflame89 — APPROVED** [review]

**Pull request merged**